ui/yarn.lock

```diff
@@ -1390,13 +1390,6 @@
     "@types/node" "*"
     jest-mock "^27.5.1"
 
-"@jest/expect-utils@^29.5.0":
```

Comment:
many of these v29 packages were being inappropriately pulled in by a `jest: "*"` request.  Not sure if this was actually causing any problems, but we can at least remove the bloat.